scripts/release.js

```diff
@@ -11,7 +17,7 @@ process.chdir(path.join(__dirname, '..'));
 
 let package_json = JSON.parse(fs.readFileSync('package.json', 'utf-8'));
 const originalVersion = package_json['version'];
-const version = originalVersion.split('.').map(x => parseInt(x));
+const version = originalVersion.split('.').map((x) => parseInt(x));
```

Comment:
nit: Why do we need (x)?

Reply:
My eslint automatically changed the `x` to `(x)`...